### PR TITLE
Feature/m2 2007 parsing activity user journey for export data

### PR DESCRIPTION
**Pull request by alex-i-e**

### CODE REVIEW

File: src/shared/utils/exportData/getJourneyCSVObject.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    flowId,
+    version,
+  } = event;
+
+  const responseValues = activityItem?.responseValues as SingleAndMultipleSelectItemResponseValues &
+    SliderItemResponseValues;
+
+  return {
+    id: event.id,
+    activity_scheduled_time: scheduledDatetime || 'not scheduled',
```
Comment (kdanilenka32) [inline src/shared/utils/exportData/getJourneyCSVObject.ts:28]: we have const for 'not sheduled'

File: src/shared/utils/exportData/getJourneyCSVObject.ts [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    activity_start_time: startDatetime,
+    activity_end_time: endDatetime,
+    press_next_time: event.type === UserActionType.Next ? event.time.toString() : '',
```
Comment (kdanilenka32) [inline src/shared/utils/exportData/getJourneyCSVObject.ts:31]: pls move event.time.toString() to var

File: src/shared/utils/exportData/prepareData.ts [hunk 3]
```diff
@@ -18,9 +24,21 @@ export const prepareData = (
   const flattenAnswers = parsedAnswers
     .flatMap((item) => item.decryptedAnswers)
     ?.filter((item) => !item.activityItem?.config?.skippableItem);
+  const flattenEvents = parsedAnswers.flatMap((item) => {
+    const decryptedAnswersObject = getDecryptedAnswersObject(item.decryptedAnswers);
```
Comment (kdanilenka32) [inline src/shared/utils/exportData/prepareData.ts:28]: It is not possible to do that in the parsedAnswers
    .flatMap topwards ?

Comment (alex-i-e) [reply src/shared/utils/exportData/prepareData.ts:28]: there are no way to do it properly in one place in flatMap

**Review by kdanilenka32 — APPROVED** [review]

**Review by IhorNesterenko99 — APPROVED** [review]

**Pull request merged**